docs/rules-engine/how-to-use/placeholders.md

```diff
@@ -109,6 +109,140 @@ Note that the URL caching mechanism is based on the url NOT 'translated', meanin
 This behavior is based on the fact that a real user rarely goes back and forth with the language update.
 
 ### Dynamic localization
+If your HTML template placeholder remains the same across different languages, you might want to consider a dynamic localization.
```

Comment:
Not sure this documentation is really linked to the rules engine or for the global usage of the placeholder mechanism.
Maybe it should be part of the placeholder documentation or at least be referred in it

Reply:
Well this documentation already contains many things on how to setup your placeholder and your PR is not yet merged.
This is a deadlock 😆 
Either we first merge this PR and you integrate this in yours or I will do it in this pr